tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -97,6 +97,9 @@ cc-qemu-tarball:
 cc-rootfs-image-tarball:
 	${MAKE} $@-build
 
+cc-sev-initrd-image-tarball: cc-sev-kernel-tarball
```

Comment:
Sorry I am being picky here. That target name is not following the "standard". We got:

rootfs-image-tarball
rootfs-initrd-tarball
cc-rootfs-image-tarball

Then I would expect this to be `cc-rootfs-initrd-sev-tarball`